css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html

```diff
@@ -68,8 +68,17 @@
 // As browsers differ in the meaning of scrollLeft when
 // in a right-to-left mode, we adjust the expectation
 // to match the semantics of scrollLeft.
-if(scroller.scrollLeft === 0)
-  expectedX = [ -box_width, -(((3*box_width - scroller_width)/2)+ (scrollbar_width/2)), -(((2*box_width)-scroller_width)+scrollbar_width)];
+if(scroller.scrollLeft === 0) {
+  // If scroll bar is on the left side, scroller.scrollLeft won't bigger then 0,
+  scroller.scrollLeft = 1000;
+  if (scroller.scrollLeft == 0) {
+    // Scroll bar is on the left side.
```

Comment:
I guess you don't need the comment here if it's explained above

Reply:
done